src/components/MultiHopTrade/hooks/useActiveQuoteStatus.tsx

```diff
@@ -79,14 +80,44 @@ export const useActiveQuoteStatus = (): QuoteStatus => {
           return 'trade.errors.quoteError'
         case ActiveQuoteStatus.NoQuotesAvailable:
           return 'trade.errors.noQuotesAvailable'
+        case ActiveQuoteStatus.SellAmountBelowMinimum:
+          return selectedSwapperName && [SwapperName.LIFI].includes(selectedSwapperName)
+            ? 'trade.errors.amountTooSmallOrInvalidTradePair'
+            : ['trade.errors.amountTooSmall', { minLimit: minimumAmountUserMessage }]
+        case ActiveQuoteStatus.SellAssetNotNotSupportedByWallet:
+          return [
+            'trade.errors.assetNotSupportedByWallet',
+            {
+              assetSymbol:
+                firstHopSellAsset?.symbol ?? translate('trade.errors.sellAssetStartSentence'),
```

Comment:
firstHopSellAsset should always be defined

```suggestion
                translate('trade.errors.sellAssetStartSentence'),
```